src/views/jobLog/index.ts

```diff
@@ -88,15 +88,23 @@ export default class JobLog implements TreeDataProvider<ProjectExplorerTreeItem>
           }
         }
       }),
-      commands.registerCommand(`vscode-ibmi-projectexplorer.jobLog.toggleFailed`, async (element: Log) => {
+      commands.registerCommand(`vscode-ibmi-projectexplorer.jobLog.showOnlyFailedJobs`, async (element: Log) => {
         const iProject = ProjectManager.get(element.workspaceFolder);
 
         if (iProject) {
           element.toggleShowFailed();
           this.refresh(element);
         }
       }),
-      commands.registerCommand(`vscode-ibmi-projectexplorer.jobLog.filterMessageSeverity`, async (element: Command) => {
+      commands.registerCommand(`vscode-ibmi-projectexplorer.jobLog.showAllJobs`, async (element: Log) => {
+        const iProject = ProjectManager.get(element.workspaceFolder);
+
+        if (iProject) {
+          element.toggleShowFailed();
```

Comment:
hopefully these never get out of sync.  It would be safer to set the desired value.  Although i see this works.